examples/analytics-notification.amp.html

```diff
@@ -66,8 +66,8 @@
   <amp-user-notification
       layout=nodisplay
       id="amp-user-notification1"
-      data-show-if-href="http://localhost:8000/api/show?timestamp=TIMESTAMP"
```

Comment:
hmm that might be a good idea, these examples were built to be copy paste-able and they aren't with localhost on the endpoints. we can just deploy this repo and it should just work (i think)